lib/internal/modules/esm/module_job.js

```diff
@@ -12,15 +12,15 @@ const resolvedPromise = SafePromise.resolve();
 class ModuleJob {
   // `loader` is the Loader instance used for loading dependencies.
   // `moduleProvider` is a function
-  constructor(loader, url, moduleProvider, inspectBrk) {
+  constructor(loader, url, moduleProvider, isMain) {
     this.loader = loader;
     this.error = null;
     this.hadError = false;
-    this.inspectBrk = inspectBrk;
+    this.isMain = isMain;
 
     // This is a Promise<{ module, reflect }>, whose fields will be copied
     // onto `this` by `link()` below once it has been resolved.
-    this.modulePromise = moduleProvider(url);
+    this.modulePromise = moduleProvider(url, isMain);
```

Comment:
is incorrect arity still a slow path

Reply:
I believe it is, yeah - not that it likely means anything in this case.